pkg/provider/loadbalancer/securitygroup/securityrule.go

```diff
@@ -107,6 +107,17 @@ func ListDestinationPrefixes(r *network.SecurityRule) []string {
 	return rv
 }
 
+func SetDestinationPrefixes(r *network.SecurityRule, prefixes []string) {
```

Comment:
Not related to the change but it should not be exported since there is no usage outside.

Reply:
Could we fix these style issues in the other PRs later? It may also need a dedicated issue to discuss these best practices, including the package structure.